docs/source/guide/secrets-manager.rst

```diff
@@ -0,0 +1,85 @@
+.. Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0
+   International License (the "License"). You may not use this file except in compliance with the
+   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+   either express or implied. See the License for the specific language governing permissions and
+   limitations under the License.
+   
+.. _aws-boto3-secrets-manager:   
+
+###############################################
+Retrieving a Secret from AWS Secrets Manager
+###############################################
+
+This Python example shows you how to retrieve the decrypted secret value from an AWS Secrets Manager secret. The secret could be created using either the Secrets Manager console or the CLI/SDK. 
+
+The code uses the AWS SDK for Python to retrieve a decrypted secret value.
+
+For more information about using an Amazon Secrets Manager, see 
+`Tutorial: Storing and Retrieving a Secret <https://docs.aws.amazon.com/secretsmanager/latest/userguide/tutorials_basic.html>`_ 
+in the *AWS Secrets Manager Developer Guide*.
+
+All the example code for the Amazon Web Services (AWS) SDK for Python is available `here on GitHub <https://github.com/awsdocs/aws-doc-sdk-examples/tree/master/python/example_code>`_.
+
+Prerequisite Tasks
+==================
+
+To set up and run this example, you must first set up the following:
+
+* Configure your AWS credentials, as described in :doc:`quickstart`.
+* Create a secret with the AWS Secrets Manager, as described in the `AWS Secrets Manager Developer Guide <https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage_create-basic-secret.html>`_
+
+Retrieve the Secret Value
+=============================================
+
+The following example shows how to:
+ 
+* Retrieve a secret value using 
+  `get_secret_value <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/secretsmanager.html#SecretsManager.Client.get_secret_value>`_.
+ 
+Example
+-------
+
+.. code-block:: python
+
+    import boto3
+    from botocore.exceptions import ClientError
+
+
+    def get_secret():
+        secret_name = "MySecretName"
+        endpoint_url = "https://secretsmanager.us-west-2.amazonaws.com"

```

Comment:
Manually setting the endpoint isn't very common. Configuring just the proper region is sufficient, so let's remove this hard coded endpoint.